cacerts/certs_test.go

```diff
@@ -186,6 +186,41 @@ func testCerts(t *testing.T, context spec.G, it spec.S) {
 			Expect(cacerts.CanonicalString("SOME VAL")).To(Equal("some val"))
 		})
 	})
+
+	context("SplitCerts", func() {
+		var dir string
+		it.Before(func() {
+			var err error
+			dir, err = ioutil.TempDir("", "multi-certs")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(os.RemoveAll(dir)).To(Succeed())
+		})
+
+		it("splits file with X certs into X new files", func() {
+			paths, err := cacerts.SplitCerts(filepath.Join("testdata", "multiple-certs.pem"), dir)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(paths).To(HaveLen(2))
+			Expect(paths).To(ConsistOf(
+				ContainSubstring("extra_0_multiple-certs.pem"),
+				ContainSubstring("extra_1_multiple-certs.pem"),
+			))
```

Comment:
`SplitCerts` is going to actually write a file, you might want to `Expect(path).To(BeARegularFile())` as well.